src/main/kotlin/app/hashers/CommitHasher.kt

```diff
@@ -35,6 +39,11 @@ class CommitHasher(private val localRepo: LocalRepo,
                    private val repo: Repo = Repo(),
                    private val api: Api,
                    private val git: Git) {
+    companion object {
+        val FACT_KEY_COMMITS_DAYWEEK = "commits-dayweek-"
+        val FACT_KEY_COMMITS_DAYTIME = "commits-daytime-"
```

Comment:
is it possible to put these into some namespace? it be nice to have something like FactKey.CommitDayWeek

Reply:
Ok.